pages/events/_id/dashboard.vue

```diff
@@ -37,20 +46,20 @@
       <TDropdown>
         <TButton
           type="context"
-          label="Add participant"
+          :label="$t('events.dashboard.participant-label')"
```

Comment:
There is a mess in naming. Let's pick one style:
1) participant-label
2) participantlabel
3) participant.label
4) participantLabel

I prefer 3d.

Reply:
or in this case it could be `events.dashboard.menu.addParticipant`

Reply:
I also only use option 3